src/Functions.php

```diff
@@ -51,3 +53,25 @@ function terminal(): Terminal
         return new Terminal;
     }
 }
+
+if (! function_exists('live')) {
```

Comment:
I believe this is not neccessary because the function is in namespace? Also it should be `function_exists('\Termwind\live')`

Reply:
Can you pull request a fix please?

Reply:
@nunomaduro Sure. Do you prefer to add namespace to check or remove the check completely?

Reply:
Add the namespace check.